.github/docs/RichTextFormattingFeature.md

```diff
@@ -0,0 +1,139 @@
+# Rich Text Formatting
+
+> ‼️ MUST ENABLE this feature for your `initiateChat()` config ["see "[Configuration](#Configuration)"]
+
+Send and receive messages with rich text formatting. By default, customer will send "text/plain" to agent.
+
+![Chat Widget with rich messaging enabled screenshot](/.github/screenshots/send-rich-messages.png)
```

Comment:
not rendering a screenshot inline. Renders a broken link

Reply:
Updated, forgot to add file to this PR

// ref: https://github.com/amazon-connect/amazon-connect-chat-interface/blob/8edb2047bd2818b2bc0bb1b2d6266ef4a7ba57c0/screenshots/send-rich-messages.png